packages/jest-cli/src/init/types.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {Config} from '@jest/types';
+
+export type ProjectPackageJson = {
+  jest?: Partial<Config.GlobalConfig>;
```

Comment:
shouldn't it be `Config.InitialOptions`?

Reply:
It should indeed!